solana-programs/staking-bridge/programs/staking-bridge/src/constant.rs

```diff
@@ -0,0 +1,16 @@
+// TODO: Use the Audius Staking Program Ethereum Address
+pub const ETH_RECIPIENT_ADDRESS_PADDED_32_BYTES: &str = "0000000000000000000000009d959Cf57D89DCf41925e19479A04E26f27563dB";
```

Comment:
need to update this address

Reply:
can we pull this from an env file so it's set properly when we deploy, but not hard-coded in?

Reply:
or maybe just rename this file constants.prod.rs?